src/HaskellCI/GitHub.hs

```diff
@@ -515,11 +526,11 @@ makeGitHub _argv config@Config {..} gitconfig prj jobs@JobVersions {..} = do
         , ghJobs = Map.fromList $ buildList $ do
             item (mainJobName, GitHubJob
                 { ghjName            = actionName ++ " - Linux - ${{ matrix.compiler }}"
-                , ghjRunsOn          = "ubuntu-18.04" -- TODO: use cfgUbuntu
+                , ghjRunsOn          = "ubuntu-" ++ ubuntuVerNum
```

Comment:
`runs-on` is in fact irrelevant, as there is container. So I'd use `ubuntu-18.04` or `ubuntu-20.04`.